lib/mongoid/contextual/memory.rb

```diff
@@ -318,7 +318,7 @@ def sort(values)
       #
       # @param [ Hash ] attributes The new attributes for the document.
       #
-      # @return [ nil, false ] False if no attributes were provided.
+      # @return [ nil | false ] False if no attributes were provided.
```

Comment:
This looks suspicious.

Reply:
Please change if needed, I just simply following what was in the docs before.